quickstart/101-azure-api-management-create/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
```

Comment:
No need to change it in these examples but you should not be using the type for the name of the resources. You should use something that represents that specific resource or example or this.